### PR TITLE
Adds VG weaponry to the server

### PR DESCRIPTION
**Pull request by TaxMaster**
Adds a bunch of ranged weaponry to the server.

Also adds toy mega-busters to the arcade machines and two vending machines inteded for adminbus/events/thunderdome.

### CODE REVIEW

**Comment by TaxMaster** [comment]
Aw shit, forgot to mention that this was non-makeshift ranged weaponry only.

File: .gitignore [hunk 1]
```diff
@@ -184,3 +184,6 @@ Temporary Items
 .apdisk
 
 *.before
+
+code/game/machinery/vending.dm
+code/game/machinery/vending.dm
```
Comment (Poojawa) [inline .gitignore]: nooooooooooooooooooooooooo

File: code/modules/projectiles/projectile/energy.dm [hunk 2]
```diff
@@ -188,3 +188,40 @@
 /obj/item/projectile/energy/tesla_revolver/Destroy()
 	qdel(chain)
 	return ..()
+
+
+obj/item/projectile/energy/plasmabolt
+	name = "plasma bolt"
+	icon_state = "plasma"
```
Comment (Poojawa) [inline code/modules/projectiles/projectile/energy.dm]: All of these could be moved to a separate .dmi and file entirely

**Comment by TaxMaster** [comment]
Ya know what?
This is so fucked, im just gonna close the PR and forget about this.

**Comment by TaxMaster** [comment]
Here is hoping its all good, now.

**Comment by TaxMaster** [comment]
Now its all good.

**Pull request merged**